README.md

```
@@ -175,3 +179,36 @@ NEXO Engine is brought to life by a dedicated team of fourth-year students from
 This project is part of our curriculum and end of studies project, showcasing our collective skills in advanced software development with modern C++.
 
 We thank Epitech for the opportunity to work on such an engaging project and for the support throughout our educational journey.
+
+## License
+
+This project is licensed under the MIT License. See the [LICENSE](LICENSE) file for details.<br/>
```

markdownlint-cli2 (0.17.2)

Inline HTML
Element: br

(MD033, no-inline-html)